tests/databases/commands_tests.py

```diff
@@ -259,7 +259,8 @@ def test_export_database_command(self, mock_g):
             "version": "1.0.0",
         }
         expected_metadata["columns"].sort(key=lambda x: x["column_name"])
-        assert metadata == expected_metadata
+        self.maxDiff = None
+        self.assertEquals(metadata, expected_metadata)
```

Comment:
Let's not change this
```suggestion
        assert metadata == expected_metadata
```